resources/metadata/mtaBuild.yaml

```diff
@@ -141,8 +141,6 @@ spec:
         type: piperEnvironment
         params:
           - name: mtarFilePath
-            fields:
-              - name: mtarFilePath
```

Comment:
How is this change related?

Reply:
this is a wrong setting which broke an integration test ...